eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -47,7 +47,7 @@ stages:
                       parameters:
                         SourceFolder: ${{parameters.ArtifactName}}
                         TargetFolder: ${{artifact.safeName}}
-                        PackageName: ${{artifact.name}}-[0-9]*.[0-9]*.[0-9]*.*.tgz
+                        PackageName: ${{artifact.name}}-[0-9]*.[0-9]*.[0-9]*.tgz
```

Comment:
I'm not sure I understand how this fixes the issue. Won't this break any beta releases?

Reply:
I think I understand it is because this isn't a regex it is wildcard syntax https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_wildcards?view=powershell-7.